src/test/resources/features/convertVideo/video.feature

```diff
@@ -0,0 +1,85 @@
+      | file              | @"template/video/demo.mp4"       |
+      | md5               | 16b258600183d1d5dc0631c82de13000 |
+      | exportFormat      | .mp4                             |
+      | name              | videoDemo                        |
+      | videoCodec        | h264                             |
+      | codec             | mp3                              |
+      | extractThumbnail  | true                             |
+      | extractMetadata   | true                             |
+
+    Then I validate the response has the "200" status code
+    And I validate that the response body should match with "video/videoSchema.json" JSON schema
+    And I validate that the response contain the following values
+      | status  | 200 |
+
+  @functional
+  Scenario: Verify that "/convertVideo" can perform "POST" request converting into MOV
+    When I send a POST request to "/convertVideo" with the following form data changing the exportFormat to MOV
+      | file              | @"template/video/demo.mp4"       |
+      | md5               | 16b258600183d1d5dc0631c82de13000 |
+      | exportFormat      | .mov                             |
+      | name              | videoDemo                        |
+      | videoCodec        | h264                             |
+      | codec             | mp3                              |
+      | extractThumbnail  | true                             |
+      | extractMetadata   | true                             |
+
+    Then I validate the response has the "200" status code
+    And I validate that the response body should match with "video/videoSchema.json" JSON schema
+    And I validate that the response contain the following values
+      | status  | 200 |
+
+  @functional
+  Scenario: Verify that "/convertVideo"  can perform "POST" request converting into GIF
+    When I send a POST request to "/convertVideo" with the following form data changing the exportFormat to GIF
+      | file              | @"template/video/demo.mp4"       |
+      | md5               | 16b258600183d1d5dc0631c82de13000 |
+      | exportFormat      | .gif                             |
+      | frames            | 24                               |
+      | controlLoop       | 0                                |
+      | duration          | 0:00:30                          |
+      | start             | 00:00:05                         |
+      | secondsToOutput   | 15                               |
+      | name              | videoDemo                        |
+      | extractThumbnail  | true                             |
+      | extractMetadata   | true                             |
+
+    Then I validate the response has the "200" status code
+    And I validate that the response body should match with "video/videoSchema.json" JSON schema
+    And I validate that the response contain the following values
+      | status  | 200 |
+  @negative
```

Comment:
Please consider adding an empty line in order to separate scenarios definition.

Reply:
I modified it according your suggestion thanks.